azure-pipelines.yml

```diff
@@ -40,22 +38,24 @@ stages:
               cmake --build $(buildOutputLocation)
             displayName: 'Build Support Tools'
             env: { TMP: $(tmpDir), TEMP: $(tmpDir) }
-          - task: BatchScript@1
-            displayName: 'Enforce clang-format'
-            timeoutInMinutes: 60
-            inputs:
-              filename: 'azure-devops/enforce-clang-format.cmd'
-              failOnStandardError: true
-              arguments: '$(buildOutputLocation)/parallelize/parallelize.exe'
-            env: { TMP: $(tmpDir), TEMP: $(tmpDir) }
-          - task: BatchScript@1
-            displayName: 'Validate Files'
-            timeoutInMinutes: 2
-            inputs:
-              filename: 'azure-devops/validate-files.cmd'
-              failOnStandardError: true
-              arguments: '$(buildOutputLocation)/validate/validate.exe'
-            env: { TMP: $(tmpDir), TEMP: $(tmpDir) }
+          - publish: '$(buildOutputLocation)/parallelize/parallelize.exe'

```

Comment:
It seems sort of overkill to publish this as an artifact and then run it after downloading it again. What was wrong with just building and running without this intermediate publishing step?

Reply:
It's split in different jobs now (to allow one to run despite the other having failed), which means they can/will end up running on a different worker which doesn't have access to the build artifacts.

Reply:
I don't see why you can't just have both the validate step and the clang-format step run no matter what.

Reply:
Fair enough, I've reverted to running them both no matter what.